kimchi/src/prover.rs

```diff
@@ -295,7 +299,9 @@ where
             .for_each(|c| absorb_commitment(&mut fq_sponge, &c.commitment));
 
         //~ 1. Compute the witness polynomials by interpolating each `COLUMNS` of the witness.
-        //~    TODO: why not do this first, and then commit? Why commit from evaluation directly?
+        //~    As mentioned above, we commit using the evaluations form rather than the coefficients
```

Comment:
NB: You'll need to update the specs (`make -C book/specifications/kimchi`) if you update a spec comment.